src/aiida/schedulers/__init__.py

```diff
@@ -13,9 +13,11 @@
 # fmt: off
 
 from .datastructures import *
+from .plugins import *
 from .scheduler import *
 
 __all__ = (
+    'BashCliScheduler',
```

Comment:
I think it make sense that `bash.py` to be  defined as a plugin, however, I don't understand why would you want it to be directly accessible through 
`schedulers.bash` instead of `schedulers.plugins.bash`. It looks a bit strange, this way.

Reply:
This is just for historical reasons. Currently all plugins that ship with `aiida-core` are in `src/aiida/schedulers/plugins` and didn't see the need to change this now or make an exception for the bash base class.

Reply:
alright, 
In that case, an initialization is missing from  `src/aiida/schedulers/plugins/__init__.py`

```python
from .bash import *

__all__ = (
    'BashCliScheduler',
)
```